### PR TITLE
Update tutorial.md

### PR DESCRIPTION
**Pull request by subinamehta**
Updated metaproteomics tutorial

### CODE REVIEW

File: topics/proteomics/tutorials/metaproteomics/tutorial.md [hunk 1]
```diff
@@ -73,7 +73,7 @@ In this tutorial, we will get the data from Zenodo: [![DOI](https://zenodo.org/b
 ## Match peptide sequences
 
 The search database labelled `FASTA_Bering_Strait_Trimmed_metapeptides_cRAP.FASTA` is the input database that
-will be used to match MS/MS to peptide sequences via a sequence database search. It is a small excerpt of the original database, which was constructed based on a metagenomic screening of the sea water samples (see [May et al. (2016)](https://www.ncbi.nlm.nih.gov/pubmed/27396978)). The full original database can be accessed from [here](https://noble.gs.washington.edu/proj/metapeptide/data/metapeptides_BSt.fasta). A contaminant database was added.
+will be used to match MS/MS to peptide sequences via a sequence database search. It is a small excerpt of the original database, which was constructed based on a metagenomic screening of the sea water samples (see [May et al. (2016)](https://www.ncbi.nlm.nih.gov/pubmed/27396978)). The full original database can be accessed from [here](https://noble.gs.washington.edu/proj/metapeptide/data/metapeptides_BSt.fasta). The contaminant database(cRAP) was merged with the original database.
```
Comment (bgruening) [inline topics/proteomics/tutorials/metaproteomics/tutorial.md]: You might want to add a space between `database` and `(cRAP)`.

File: topics/proteomics/tutorials/metaproteomics/tutorial.md [hunk 2]
```diff
@@ -96,12 +96,12 @@ The created dataset collection of the three *MGF files* in the history is used a
...
 >    Section **Search Engine Options**:
 >
->    - **B-Search Engines**: `X!Tandem`
+>    - **Search Engines**: `X!Tandem`
 >
 >    > ### {% icon comment %} Comment
 >    >
->    > The section **Search Engine Options** contains a selection of sequence database searching
->    > programs that are available in SearchGUI. Any combination of these programs can be used for
+>    > This section **Search Engine Options** contains a selection of sequence database searching
```
Comment (bgruening) [inline topics/proteomics/tutorials/metaproteomics/tutorial.md]: Any reason to replace `The`. It sounds better to me :) But I have usually no clue about english :)

File: topics/proteomics/tutorials/metaproteomics/tutorial.md [hunk 3]
```diff
@@ -434,7 +434,7 @@ community based on expressed microbial proteome.
 In the following chapter, a functional analysis will be performed using the **UniPept** application `pept2prot` in order to match the list of peptides with the correlated Gene Ontology terms.
 This allows to get an insight of the **biological process**, the **molecular function** and the **cellular component** related to the sample data.
 
-> ### {% icon comment %} Gene Ontology Consortium
+> ### {% icon comment %} Gene Ontology(GO) Consortium
```
Comment (bgruening) [inline topics/proteomics/tutorials/metaproteomics/tutorial.md]: space missing?

**Comment by bgruening** [comment]
Thanks @subinamehta!!!

File: topics/proteomics/tutorials/metaproteomics/tutorial.md (hunk 2 above)
Comment (subinamehta) [reply topics/proteomics/tutorials/metaproteomics/tutorial.md]: You're right, The does sound better!!!

File: topics/proteomics/tutorials/metaproteomics/tutorial.md (hunk 1 above)
Comment (subinamehta) [reply topics/proteomics/tutorials/metaproteomics/tutorial.md]: I will add the space

**Comment by subinamehta** [comment]
I can make additional changes!

**Comment by shiltemann** [comment]
thanks a lot @subinamehta!

**Pull request merged**